macro/build_geometry.C

```diff
@@ -127,7 +127,10 @@ void build_geometry(FairRunSim* run = nullptr)
   // beam pipe
   if (isActivated("PIPE")) {
 #ifdef ENABLE_UPGRADES
-    run->AddModule(new o2::passive::Pipe("PIPE", "Beam pipe", 1.6f, 0.05));
+    if (isActivated("IT3") || isActivated("IT4"))
```

Comment:
So it looks like we were using the wrong pipe for Run3 (because ENABLE_UPGRADES is switched on by default for CVMFS). We should re-discuss this macro. Potentially we should just get rid of it completely.

As a side note, please note that coding conventions require '{}' for if - else bodies.

Reply:
Yes, that's what I am trying to fix here.

We had discussed in the past that it would be useful to have the option of not compiling the upgrades, but in this case enabling them should not change the behaviour. If we change our mind and decide to always compile them, that's also fine with me.